source/documentation/administration_guide/topics/Replacing_the_Manager_CA_Certificate.adoc

```diff
@@ -8,6 +8,12 @@ You can configure your organization's third-party CA certificate to authenticate
 
 Third-party CA certificates are not used for authentication between the {engine-name} and hosts or for link:{URL_rest_api_doc}index#types/image_transfer[disk transfer URLs]. These HTTPS connections use the self-signed certificate generated by the {engine-name}.
 
+[IMPORTANT]
+====
+When you switch to a custom HTTPS certificate, you must use your own CA certificate distribution to make that certificate available on clients.
```

Comment:
Are these two sentences connected? Is there a reason one begins with "when" and the other begins with "if"? 
If the sentences are not connected, consider breaking this into two notes.

Reply:
The second line is an added note to the same general action, but it is a sub-scenario, not the main scenario

Reply:
Then consider something like "If you switch to a custom HTTPS certificate and integrating with Red Hat Satellite, you need to.."